news/lykke2_08-09-17.md

```diff
@@ -1,10 +1,11 @@
 ### Project: [Lykke](../projects/lykke.md)
 ### Date: 08 September 2017
 ## Summary
+  
 Lykke announces the grand opening of Lykke de México S.A. de C.V., a Mexican subsidiary designed to bring the unique Lykke trading experience to the Latin American market.
```

Comment:
...to the Latin American market.[double space]
...Lykke de México.[double space]